src/main/resources/db/migration/ddl/V1_0__baseline.sql

```diff
@@ -126,3 +126,28 @@ create table shedlock
 create index EVENTS_EVENTABLE_ID_IDX on events (eventable_id);
 
 create index JOURNEYS_MOVE_ID_IDX on journeys (move_id);
+
+CREATE TABLE SPRING_SESSION
```

Comment:
So I had to bring this into here as part of the update to latest Spring Boot.  It was not playing nicely with the Flyway DB migration tool.  In short all of the DB tables needed are now included in the baseline script.